README.md

```diff
@@ -1108,7 +1117,8 @@ where:
 
 * `space_name` (`string`) - name of the space
 * `opts`:
-  * `timeout` (`?number`) - `vshard.call` timeout (in seconds)
+  * `timeout` (`?number`) - `vshard.call` timeout and vshard master
+    discovery timeout (in seconds)
```

Comment:
Since we've updated the descriptions, let's describe default everywhere

Reply:
Done, added `, default value is 2`